server/sonar-server/src/main/java/org/sonar/server/component/es/ProjectMeasuresIndexer.java

```diff
@@ -68,23 +70,23 @@ public void deleteProject(String uuid) {
 
   private long doIndex(BulkIndexer bulk, long lastUpdatedAt, @Nullable String projectUuid) {
     try (DbSession dbSession = dbClient.openSession(false)) {
-      ProjectMeasuresResultSetIterator rowIt = ProjectMeasuresResultSetIterator.create(dbClient, dbSession, lastUpdatedAt, projectUuid);
+      ProjectMeasuresIndexerIterator rowIt = ProjectMeasuresIndexerIterator.create(dbSession, lastUpdatedAt, projectUuid);
```

Comment:
`rowIt` should be moved to try-with-resource, so that it does not need to be closed at line 75 and it does not leak on exceptions. Note that I know that there is the same issue on other indexers !